SwiftCompilerSources/Sources/SIL/Function.swift

```diff
@@ -145,6 +145,14 @@ final public class Function : CustomStringConvertible, HasShortDescription, Hash
 
   public enum SerializedKind {
     case notSerialized, serialized, serializedForPackage
+
+    public var bridged: BridgedFunction.SerializedKind {
```

Comment:
If it's avoidable, the `bridged` properties should not be exposed as public properties. They are an internal implementation detail to the implementation. For example, passes should never access `bridged`. 

In a future, far, far away, we might implement SIL itself in swift and then `bridged` goes away completely.

But: often it is required to make `bridged` public, because e.g. `Context` needs to access it.